addons/web/static/tests/core/dialog_tests.js

```diff
@@ -296,4 +296,44 @@ QUnit.module("Components", (hooks) => {
             "UI owner should be reset to the default (document)"
         );
     });
+
+    // This test is skipped because we cannot drag and drop in test
+    QUnit.skip("dialog can be moved", async (assert) => {
```

Comment:
What's the point of these tests? Why can't we d&d in tests?

Reply:
I think I've found why it didn't work.
The position I give to the dragAndDrop function seems incorrect.